e2e/cypress/integration/ui/settings/05_project_mgmt.spec.ts

```diff
@@ -207,6 +208,30 @@ describeIfIAMV2p1('project management', () => {
 
     cy.get('app-project-list chef-button').contains('Delete Project').click();
 
-    cy.get('app-project-list chef-tbody chef-td').contains(projectID).should('not.exist');
+    // empty UI should show up so entire table will be missing
```

Comment:
What if other projects have been created? Let's keep it the way it was to account for that.